app/mailers/teacher_mailer.rb

```diff
@@ -71,6 +71,21 @@ def further_information_reminder
     )
   end
 
+  def initial_checks_passed
+    @teaching_authority_certificate =
+      region.teaching_authority_certificate.presence ||
+        region.country.teaching_authority_certificate
+    @teaching_authority_name =
+      region.teaching_authority_name.presence ||
+        region.country.teaching_authority_name
```

Comment:
There are some helpers which can do this for us, we could use them directly in the view:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/2938ba3a3916335f944b6ce288a4bceba5bacafb/app/helpers/region_helper.rb#L15-L18